buildSrc/src/main/java/io/datahubproject/OpenApiEntities.java

```diff
@@ -117,14 +122,19 @@ public ObjectNode entityExtension(List<ObjectNode> nodesList, ObjectNode schemas
         return componentsNode;
     }
 
-    private static String toUpperFirst(String s) {
-        return s.substring(0, 1).toUpperCase() + s.substring(1);
+    public static String toUpperFirst(String s) {
+        if (s.length() > 2 && s.substring(2, 3).equals(s.substring(2, 3).toUpperCase())) {
```

Comment:
If the third character is upper case then the first two should be upper case? Maybe worth a comment here explaining why

Reply:
Basically `mlModel -> MLModel` instead of `mlModel -> MlModel`